configure.ac

```diff
@@ -312,13 +312,12 @@ AC_ARG_WITH([x-toolkit],[AS_HELP_STRING([--with-x-toolkit=KIT],
  [use an X toolkit (KIT one of: yes or gtk, gtk2, gtk3, no)])],
 [	  case "${withval}" in
 	    y | ye | yes )	val=gtk ;;
-	    n | no )		val=no  ;;
 	    g | gt | gtk  )	val=gtk ;;
 	    gtk2  )	val=gtk2 ;;
 	    gtk3  )	val=gtk3 ;;
 	    * )
```

Comment:
Why are we dropping the `no` support? Emacs should be able to build with a GUI.

Reply:
* `--with-x-toolkit=no` means building with X raw API (There is no x toolkit such as lucid).
* `--with-x-window=no` means building without GUI (That will be an terminal Emacs).

Reply:
:+1: